docs/tutorials/qml/ml_tools.md

```diff
@@ -140,18 +142,36 @@ def loss_fn(model: torch.nn.Module, data: torch.Tensor) -> tuple[torch.Tensor, d
     loss = criterion(out, y)
     return loss, {}
 
+# Here we use the validation criterion that checks whether the validation loss in the current
+# iteration has decreased compared to the lowest validation loss from all previous iterations.
+def validation_criterion(
```

Comment:
Typically it's best to avoid long comments in a documentation snippet. You can break the snippet in multiple ones and add this text as normal documentation description in between.

Reply:
Thanks for this suggestion, done!